utils/check_code_format.sh

```diff
@@ -18,14 +18,16 @@
 #
 # This script assumes to be invoked at the project root directory.
 
-FILES_TO_CHECK=$(git diff --name-only master | grep -E ".*\.(cpp|cc|c\+\+|cxx|c|h|hpp)$")
+BASE_BRANCH=${BASE_BRANCH:-master}
```

Comment:
Seems ok, but why not make this an argument to the script?

Reply:
I did consider it. No real reason beyond wondering whether this would be the preferred way. I've updated the PR.